source/adapter.js

```diff
@@ -254,13 +254,71 @@ AdapterJS.maybeFixConfiguration = function (pcConfig) {
 AdapterJS.addEvent = function(elem, evnt, func) {
   if (elem.addEventListener) { // W3C DOM
     elem.addEventListener(evnt, func, false);
-  } else if (elem.attachEvent) {// OLD IE DOM 
+  } else if (elem.attachEvent) {// OLD IE DOM
     elem.attachEvent('on'+evnt, func);
   } else { // No much to do
     elem[evnt] = func;
   }
 };
 
+AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink) {
```

Comment:
If you don't use it I think we should move it back for now.
Putting a function in a public namespace means that we have to maintain it in the future.